docs/troubleshoot/cli/known-cli.md

```diff
@@ -12,27 +12,26 @@ After you install Zowe CLI, and the installation appears to complete successfull
 
 - `zowe config init`
 - `zowe config secure`
-- `zowe profiles create`
 - Most Zowe commands that access your mainframe environment
 
 This behavior occurs under the following conditions:
 
-- npm version 8.11.0 or 8.12.0 is running on your computer.
+- npm version 8.11.0 or 8.12.0 is running on your computer. **[what should the versions be for V3?]**
```

Comment:
@adam-wolfe - please respond

Reply:
I would expect most users to be on npm v10.x.x, but since this issue is specific to npm 8.11.0 & 8.12.0, I would leave this as-is.